src/main/plugins/org.dita.base/init/startup/topics/welcome.dita

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE topic PUBLIC "-//OASIS//DTD DITA Topic//EN" "topic.dtd">
+<topic id="welcome">
+  <title>My first topic</title>
+  <body>
+    <p>Welcome to your first <ph keyref="product"/>.</p>
+    <p>You can add images:</p>
+    <image href="../images/welcome.png" id="image_j3n_cc1_bpb" placement="break"/>
```

Comment:
Missing image reference

Reply:
This is not meant to actually be something we use as is, because we'd need to create some image or use an existing image that has a suitable license. This is just a in progress example of what we could have. The question what should init project contain.